apis/config/v1alpha1/cluster_colocation_profile_types.go

```diff
@@ -26,19 +26,17 @@ import (
 
 // ClusterColocationProfileSpec is a description of a ClusterColocationProfile.
 type ClusterColocationProfileSpec struct {
-	// NamespaceSelector decides whether to mutate/validate Pods on an object based
-	// on whether the namespace for that object matches the selector.
+	// NamespaceSelector decides whether to mutate/validate Pods based
+	// on whether the namespace for that matches the selector.
 	// Default to the empty LabelSelector, which matches everything.
 	// +optional
 	NamespaceSelector *metav1.LabelSelector `json:"namespaceSelector,omitempty"`
 
-	// ObjectSelector decides whether to mutate/validate Pods based on if the
+	// Selector decides whether to mutate/validate Pods based on if the
 	// Pod has matching labels.
-	// Use the object selector only if the profile is opt-in, because end
-	// users may skip the admission webhook by setting the labels.
 	// Default to the empty LabelSelector, which matches everything.
 	// +optional
-	ObjectSelector *metav1.LabelSelector `json:"objectSelector,omitempty"`
+	Selector *metav1.LabelSelector `json:"selector,omitempty"`
```

Comment:
I think `PodSelector` might be more easy to understand. WDYT.

Reply:
Selector is K8s API design idiom. Like StatefulSet.Spec.Selector or Deployment.Spec.Selector